go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/fullstorydev/grpcurl
 
+go 1.14
```

Comment:
Mind removing this? We _just_ finished a migration off of App Engine (go111 runtime), but we haven't yet hoisted ourselves off of Go 1.11. And 1.11 chokes on these.

Reply:
Hmmm, well CI is green, so I guess I am wrong. Maybe it was fixed in a later point release of Go 1.11...